docs/ReferenceInput.md

```diff
@@ -166,32 +166,6 @@ You can filter the query used to populate the possible values. Use the `filter`
 
 **Note**: When users type a search term in the `<AutocompleteInput>`, this doesn't affect the `filter` prop. Check the [Customizing the filter query](#customizing-the-filter-query) section below for details on how that filter works.
 
-## `format`
```

Comment:
Moved below as " Transforming The Input Value" because the `format` prop doesn't actually exist in this component.